src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -193,6 +201,24 @@ bool icm426xxSpiAccDetect(accDev_t *acc)
 
 static aafConfig_t getGyroAafConfig(void);
 
+static void turnGyroAccOff(const extDevice_t *dev)
+{
+    spiWriteReg(dev, ICM426XX_RA_PWR_MGMT0, ICM426XX_PWR_MGMT0_GYRO_ACCEL_MODE_OFF);
+    delay(15);
+}
+
+// Turn on gyro and acc on in Low Noise mode
+static void turnGyroAccOn(const extDevice_t *dev)
+{
+    spiWriteReg(dev, ICM426XX_RA_PWR_MGMT0, ICM426XX_PWR_MGMT0_TEMP_DISABLE_OFF | ICM426XX_PWR_MGMT0_ACCEL_MODE_LN | ICM426XX_PWR_MGMT0_GYRO_MODE_LN);
+    delay(15);
```

Comment:
As above

Reply:
This one is based on section 14.36 in the data sheet for the ICM-42688-P

> When transitioning from OFF to any of the other modes, do not issue any register writes for 200μs.

However, 15 ms is excessive (once again, extra caution, 15 was selected due to copy pasting the value from other parts of the driver).
I'll try setting it to 1.

Reply:
This delay is decreased to 1 ms